app/Jobs/ProcessDocuSignWebhook.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Jobs;
+
+use App\Models\DocuSignEnvelope;
+use Spatie\WebhookClient\Jobs\ProcessWebhookJob;
+
+class ProcessDocuSignWebhook extends ProcessWebhookJob
```

Comment:
This will get more complicated over time - in particular, we're not storing the documents themselves within Apiary at this point. That's fine(ish) for membership agreements but will need to be added for travel.

Reply:
Documents are now being received via webhook and stored on disk.